src/data/mentorship-timeline-4.js

```diff
@@ -0,0 +1,34 @@
+export default [
+  {
+    date: "May 2023",
+    title: "Mentorship Applications Open",
+    description: <>Apply to be a mentor for Techladies Mentorship</>,
+  },
+  {
+    date: "June 2023",
```

Comment:
I was under the impression that we'd be doing these in parallel. Has that changed or was that more of our internal strategy?

Reply:
it'll be good to have a few mentors locked in before mentee application. but it's in parallel still as our forms will be opened.